examples/Basic/Readme.md

```diff
@@ -1,12 +1,15 @@
-Dropzone with default properties and displays list of the dropped files.
+By default, Dropzone just renders provided children without applying any styles.  
+
+This example renders a list of the dropped files.
 
 ```
 class Basic extends React.Component {
-  constructor() {
-    super()
+  constructor(props) {
+    super(props)
     this.state = { files: [] }
+    this.onDrop = this.onDrop.bind(this)
   }
-
+  
```

Comment:
How new style do you prefer here?

Reply:
What?

Reply:
I mean, code style (state to class, new style binding).